controllers/messages/config_table.yaml

```diff
@@ -13,3 +13,6 @@ columns:
         title: From
     to:
         title: To
+    found:
+        title: Found
```

Comment:
We should use a translation string here i.e. rainlab.translate::lang.messages.found_title

Reply:
Translations are not necessary because the actual title comes from [this line](https://github.com/multiwebinc/translate-plugin/blob/7fc5503dc78ab8045ae202b378982d7d288920d7/controllers/messages/_table_headers.htm#L104). I am assuming they are only used here as a fallback in case something down the line should fail for whatever reason. This line can actually be commented out and it will still work.